content/guides/api-overview.md

```diff
@@ -0,0 +1,75 @@
+# Smily Channel API Overview
+
+1. TOC
+{:toc}
+
+## What is Smily Channel API?
+
+The channel API helps channel partners to integrate with our API by providing “shortcuts” so that partners can pull off the required data they need without having to combine too many requests from our API.
+
+## Who is Smily Channel API for?
+
+The Smily Channel API is designed to be useful for a variety of platforms. It's particularly valuable for:
+
+  - **Online Travel Agencies (OTAs)**
+  - **Marketplaces**
+  - **Tour Operators**
+  - **Niche Channels**
+
+**Create Your Own Portal:** Additionally, the Smily offers the flexibility to build entire Marketplaces or Niche Channels according to your vision. Explore this capability further on our [Your Own Portal](https://www.smily.com/en/your-own-portal) page.
+
+## Why would I use Smily Channel API?
+
+The Smily Channel API presents an optimal way to easily onboard new properties onto your system. Designed with our partners in mind, this API simplifies the integration journey.
+
+It's important to highlight that our more extensive feature set is available through the [BookingSync Universe API ](https://developers.bookingsync.com/). While the Smily Channel API offers speedy integration, unlocking the full range of features in the BookingSync Universe API might take more time and expertise from skilled engineers.
```

Comment:
BookingSync Universe API  - as far as I remember - this is our internal naming. May be better just to call it like our Public Api?

PS there is link to universe api - https://developers.bookingsync.com/guides/bookigsync-universe-api-early-beta/ but this is not what you sent